src/components/LHNOptionsList/OptionRowLHNData.tsx

```diff
@@ -1,60 +1,13 @@
 import {deepEqual} from 'fast-equals';
-import PropTypes from 'prop-types';
 import React, {useEffect, useMemo, useRef} from 'react';
-import _ from 'underscore';
-import participantPropTypes from '@components/participantPropTypes';
-import transactionPropTypes from '@components/transactionPropTypes';
 import * as ReportActionsUtils from '@libs/ReportActionsUtils';
 import SidebarUtils from '@libs/SidebarUtils';
 import * as TransactionUtils from '@libs/TransactionUtils';
-import reportActionPropTypes from '@pages/home/report/reportActionPropTypes';
-import * as Report from '@userActions/Report';
+import * as ReportLib from '@userActions/Report';
```

Comment:
Why did you rename it?

Reply:
Because I am importing `Report` type and it was conflicting

Reply:
@kubabutkiewicz Maybe we can import types like `import * as OnyxTypes from './types/onyx';`, we do this in similar cases, but it's up to you